index.html

```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
             </div>
         </section>
       <form action="https://formspree.io/f/xlezlkyw" method="post">
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="forms">  
+        <p class="mine">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="inputs">
         <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
         <input name="email" type="email" placeholder="Email address" required>
         <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
         <button class="get" type="submit">Get in touch</button>
+      </div>
```

Comment:
- [ ] <b>REQUIRED:</b> Like mentioned by the previous reviewer, you need to make sure that your form looks more like the provided design. Firstname and Lastname should be in two different field and also Kindly make sure to style them like how they are styled in Figma design. Also your button text is overlapping. 

| Figma      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/94551484/186989371-4ad8bbc7-b9be-48a5-b62a-3a836ab5bc35.png) | ![Screenshot](https://user-images.githubusercontent.com/45217364/187026986-37d3d84d-8fed-42a7-a022-f00258e30716.png) |